.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
- [ ] Please make sure you have added a descriptive Readme file added to your project. You can use the templates given by Microverse.